src/test/java/com/google/javascript/clutz/partial/static_method.d.ts

```diff
@@ -0,0 +1,19 @@
+declare namespace ಠ_ಠ.clutz {
+  class module$exports$foo$bar$Class extends module$exports$foo$bar$Class_Instance {
+    //!! The parameter type annotations match the (renamed) class name
```

Comment:
Won't the test suite eat comments in here?

Reply:
Say more?

Reply:
Isn't this file effectively a golden generated by the test suite?  E.g. if you edit static_method.js, to rebaseline, don't you copy the output over this file?

Reply:
Good point - removed.